ACKImagePicker/ViewControllers/ProgressViewController.swift

```diff
@@ -15,6 +15,12 @@ final class ProgressViewController: UIViewController {
         }
     }
 
+    var progressTintColor: UIColor! {
```

Comment:
do we want this to be impliciltly unwrapped? I think better solution here is to make it optional and add `get`, `set`

Reply:
I will make it optional, but I cannot use `get` and `set` since this is `UIViewController` and this property can be set before `loadView` is called. The implicitly unwrapped optional was used there to copy the interface of `progressView`